doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+.. _quantization:
+
+Quantization
+============
+
+nGraph Quantization 
+-------------------
+
+Quantization refers the process of reducing the number of bits that represent a
+number. In the context of deep learning, weights and activations can be
```

Comment:
Use the `:abbr:` syntax since you refer to deep learning as DL later in the file.  

> In the context of :abbr:`Deep Learning (DL)` ...